File: Lab5B/Lab 20/Employee.cs
using System;

namespace Lab_20
{
    class Employee
    {
        public Employee(string _first, string _last, decimal _sal)
        {
            FirstName = _first;
            LastName = _last;
            YearlySalary = _sal;
        }
        public void GiveRaise()
        {
            YearlySalary *= 1.10M;
        }
        public void PrintData()
        {
            Console.WriteLine($"{FirstName} {LastName}: Yearly Salary: {YearlySalary:C}");
        }

        private string FirstName {get; set;}
        private string LastName {get; set;}
        private decimal YearlySalary {get; set;}
    }
}

File: Lab5A/src/lab/employee/test/Employee.java
package lab.employee.test;

public class Employee 
{
	private String firstName;
	private String lastName; 
	private double monthly; 
	
	public Employee (String first, String last, double month){
		firstName = first; 
		lastName = last; 
		monthly = month; 

		if (month < 0.0)
			monthly = 0.0;
	}
	public void setFirstName (String first){
		firstName = first;	
	}
	public String getFirstName (){
		return firstName;
	}
	public void setLastName (String last){
		lastName = last;
	} 
	public String getLastName (){
		return lastName;
	} 
	public void setMonthlySalary (double month){
		monthly = month;
	} 
	public double getMonthlySalary (){
		return monthly;
	} 
	public double getYearlySalary(){
		double yearly = monthly * 12;
		return yearly;
	} 
	public double getRaiseSalary(){
		double raise =  monthly * 0.1 ;
		double raiseSalary = ( monthly + raise ) * 12;
		return raiseSalary;
	} 	
} 
